core/simpleproducer.go

```diff
@@ -123,9 +125,28 @@ func (prod *SimpleProducer) Configure(conf PluginConfigReader) error {
 	dropStreamID := StreamRegistry.GetStreamID(conf.GetString("DropToStream", DroppedStream))
 	prod.dropStream = StreamRegistry.GetRouterOrFallback(dropStreamID)
 
+	// Simple health check for the plugin state
+	//   Path: "/<plugin_id>/SimpleProducer/pluginstate"
+	prod.AddHealthCheckAt("/SimpleProducer/pluginState", func()(code int, body string) {
```

Comment:
Remove SimpleProducer from the path because this creates a semantic link to the actual implementation which might get problematic in the future.